docs/backup-recovery/README.md

```diff
@@ -0,0 +1,146 @@
+
+### States ###
+
+- BackupStatePreparingToStart - is the initial state all clusters are automatically assigned to on the first reconciliation pass of the TALO CR. Upon entry TALO deletes spoke backup namespace and hub view resources that might have remained from the prior incomplete attempts.
+- BackupStateStarting - is the state for creation of the backup job pre-requisites and the job itself
+- BackupStateActive - the job is in "Active" state
+- BackupStateSucceeded - a final state reached when the backup job has succeeded
+- BackupStateTimeout - a final state meaning that artifact backup has been partially done
+- BackupStateError - a final state reached when the job ends with a non-zero exit code
+
+### On the spoke ###
+
+The backup workload generates an utility called `upgrade-recovery.sh` in the recovery partition or at the recovery folder at `/var/recovery` and takes the pre-upgrade backup. In addition, the active OS deployment is pinned using ostree and the standby deployments are removed.
+
+#### Procedure end options ####
+
+- Success (“Completed”)
+- Failure due to timeout (“DeadlineExceeded”)  
+  - the procedure did not complete due to a timeout and can be continued
+  - Unrecoverable error (“BackoffLimitExceeded”) - user intervention is desired to determine the failure reason
+
+
+## Recovery from Upgrade Failure
+
+### Platform Rollback
+
+Platform rollback, if needed, is handled by issuing the `rpm-ostree rollback -r` command. Not all upgrades will include
+an update to the platform OS, however, so this step will not always be required.
+
+When the backup was taken, the active deployment was pinned and standby deployments removed. Use the
+`ostree admin status` command to see the current state.
+
+|Deployment Status|Rollback Action|
+|-----------------|---------------|
+|Single deployment, pinned|Platform OS unchanged, no action required|
+|Standby deployment pinned, flagged as "rollback"|Trigger rollback with `rpm-ostree rollback -r` command|
+|Multiple standby deployments, pinned deployment is not "rollback"|1. Delete unpinned standby deployments with `ostree admin undeploy 1`, until pinned deployment is "rollback".<br>2. Trigger rollback with `rpm-ostree rollback -r` command|
+
+### Recovery Utility
+
+The upgrade recovery utility is generated when taking the backup, before the upgrade starts, written as `/var/recovery/upgrade-recovery.sh`.
+
+* The first phase of the recovery shuts down containers and cleans up, and restore files from the backup partition/folder to the targeted directories. Afterwards, it prompts for a rebooting the node with `systemctl reboot`.
+
+* The second stage restores cluster database and related files from the backup, relaunches containers and trigger required redeployments as per cluster restore procedure.
+
+
+> **WARNING**: Should the recovery utility fail, the user can retry with the `--restart` option:<br>
+`/var/recovery/upgrade-recovery.sh --restart`
+
+
```

Comment:
Do you think it will be useful to include the troubleshooting section we have documented?

Reply:
As mentioned earlier, we are reducing/focusing the scope of the Readme. For now, troubleshooting can reside at design doc and will be added by Doc team for downstream documentation. Thanks